src/envoy/utils/utils.cc

```diff
@@ -70,6 +76,21 @@ bool GetIpPort(const Network::Address::Ip* ip, std::string* str_ip, int* port) {
   return false;
 }
 
+bool GetDestinationUID(const envoy::api::v2::core::Metadata& metadata,
+                       std::string* uid) {
+  const auto filter_it = metadata.filter_metadata().find(kPerHostMixer);
+  if (filter_it == metadata.filter_metadata().end()) {
+    return false;
+  }
+  const auto fields_it =
```

Comment:
can we do following to make this code more readable

const Struct& struct_pb = filter_it->second;

Reply:
Done